requirements.txt

```diff
@@ -1,4 +1,4 @@
-Django==1.6.11
+Django==1.7
 Jinja2==2.7.3
 MarkupSafe==0.23
 mysqlclient==1.4.6
```

Comment:
South is deprecated from Django 1.7. Can you try to remove https://github.com/haystack/murmur/blob/4f1a3f62e21d5fe360507dad0c51ae6743df13cf/requirements.txt#L5 line and see if it is still working?